specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -0,0 +1,585 @@
+        "operationId": "DataRegistries_List",
+        "x-ms-examples": {
+          "List all previously registered data registries": {
+            "$ref": "./examples/DataRegistries_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request completed successfully. The response body contains a list of all the previously registered data registries.",
+            "schema": {
+              "$ref": "#/definitions/DataRegistryList"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../Common/preview/1.0/common.json#/definitions/MapsErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/dataRegistries/{udid}": {
+      "get": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of Data Registry APIs is bound to the region of your maps account. Data Registry Service is not available for global maps account.\n\nThis API allows the caller to fetch the data registry of a given udid registered previously using the [Register or Replace API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register-or-replace). If Azure Maps cannot access the underlying content or integrity check is not passed, `status` will be `Failed` and error detail is returned.\n\n## Submit Get Data Registry Request\n\nTo fetch the data registry of a given udid you will issue a `GET` request with no additional parameters.",
+        "operationId": "DataRegistries_GetDataRegistry",
+        "x-ms-examples": {
+          "Get the data registry of a given valid udid": {
+            "$ref": "./examples/DataRegistries_GetDataRegistryValid.json"
+          },
+          "Get the data registry of a given invalid udid": {
```

Comment:
maybe change the title to something like "Get the data registry with Failed status".
invalid udid sounds like udid is invalid. (not eixst or malformed)

Reply:
Fixed!